module_intros/order.Steinhardt.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<p>Maybe add an extra sentence or two explaining the various extra modes since you already discuss Wl and average? Might be helpful.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>